### PR TITLE
Fun with docker

### PR DESCRIPTION
**Pull request by fisx**
- eliminate reference of images on hub.docker.com and don't build those with travis any more.
- tweak integration.sh
- [most importantly] tweak Dockerfile.builder so that it can be used to run `make integration` on the command line inside the container.  provide target for this in top-level Makefile.

### CODE REVIEW

File: build/alpine/Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-DOCKER_USER   ?= wireserver
+DOCKER_USER   ?= wire
```
Comment (jschaul) [inline build/alpine/Makefile]: For consistency: `DOCKER_USER      ?= quay.io/wire`

File: Makefile [hunk 2]
```diff
@@ -74,6 +74,14 @@ docker-exe-%:
 docker-service-%:
 	$(MAKE) -C services/"$*" docker
 
+DOCKER_DEV_NETWORK := --net=host
+DOCKER_DEV_VOLUMES := -v `pwd`:/src/wire-server
+DOCKER_DEV_IMAGE   := wire/alpine-builder:local
```
Comment (jschaul) [inline Makefile]: For consistency: `DOCKER_DEV_IMAGE   := quay.io/wire/alpine-builder:local`

File: Makefile [hunk 3]
```diff
@@ -74,6 +74,14 @@ docker-exe-%:
...
+.PHONY: run-docker-dev
+run-docker-builder:
+	docker run -it $(DOCKER_DEV_NETWORK) $(DOCKER_DEV_VOLUMES) --rm $(DOCKER_DEV_IMAGE) /bin/bash || \
+	( echo "$(DOCKER_DEV_IMAGE) not found.  building locally.  hit ^C to interrupt." && make -C build/alpine builder )
```
Comment (jschaul) [inline Makefile]: `echo "..." && make ...` && `docker run -it $(DOCKER_DEV_NETWORK) $(DOCKER_DEV_VOLUMES) --rm $(DOCKER_DEV_IMAGE) /bin/bash` (add the actual command one more time after being done building the alpine-builder ?)

**Comment by fisx** [comment]
thanks, fixed everything (i think).  except the concourse resource exhaustion :-).

have you tried this locally yet?

**Pull request merged**